client/src/index.scss

```diff
@@ -1,14 +1,10 @@
-// Override default Bootstrap variables before the import
-$body-bg: #fff;
-$blue: #425af2;
-
-// Import Bootstrap and its default variables
-@import "~bootstrap/scss/bootstrap.scss";
+// Import Theme Style
+@import "./styles/theme.scss";
 
 body {
   margin: 0;
-  font-family: "Poppins", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
-    "Droid Sans", "Helvetica Neue", sans-serif;
+  background: $body-bg;
+  font-family: $font-family;
```

Comment:
yes can probably remove these things and move them to Styled Components theme

Reply:
You mean using ThemeProvider?

Reply:
yes I guess replace what we had before to use https://styled-components.com/docs/advanced#theming